### ISSUE TITLE
Bitpanda error  fetchOpenOrders(), fetchOrder()

### ISSUE THREAD

**Issue by Doccos** [ccxt/ccxt#13463]
On bitpanda i get follow error when i try to get some informations about trades:

UncaughtException: TypeError: Cannot read properties of undefined (reading 'rate')

- OS: ubuntu 22.04
- Programming Language version: Node.js v16.15.0
- CCXT version: 1.84.15

```
const exchange = new ccxt["bitpanda"]({
    apiKey: "i am the api key"
});

;(async ()=> {
    await exchange.fetchOpenOrders("BTC/EUR");

    //SAME WITH await this.exchange.fetchOrder("eedhhda5-6f75-1234-5678-14c8adaxfa1a","BTC/EUR")
})();
```

```
UncaughtException: TypeError: Cannot read properties of undefined (reading 'rate')
    at bitpanda.safeOrder (/var/nodejs/trade-ts/node_modules/ccxt/js/base/Exchange.js:2119:80)
    at bitpanda.parseOrder (/var/nodejs/trade-ts/node_modules/ccxt/js/bitpanda.js:1400:21)
    at bitpanda.fetchOrder (/var/nodejs/trade-ts/node_modules/ccxt/js/bitpanda.js:1586:21)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Timeout._onTimeout (/var/nodejs/trade-ts/compiled/lib/hive/CctxHive.js:26:48)

```

**Comment by kroitor** [comment]
@Doccos please, paste your verbose output:

- https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

```JavaScript

console.log ('CCXT Version:', ccxt.version)

const exchange = new ccxt["bitpanda"]({
    apiKey: "i am the api key"
});

;(async ()=> {

    await exchange.loadMarkets()
    exchange.verbose = true  // <<<<<<<<<<<<<< after loading the markets
    
    
    await exchange.fetchOpenOrders("BTC/EUR");

    //SAME WITH await this.exchange.fetchOrder("eedhhda5-6f75-1234-5678-14c8adaxfa1a","BTC/EUR")
})();
```

**Comment by Doccos** [comment]
Hello,
Now it works with exchange.fetchOpenOrders but not with exchange.fetchCloseOrders
mabye the error comes if there no orders

the verbose output (with exchange.fetchCloseOrders("BTC/EUR")):
```CCXT Version: 1.84.15
fetch Request:
 bitpanda GET https://api.exchange.bitpanda.com/public/v1/account/orders?instrument_code=BTC_EUR&with_cancelled_and_rejected=true 
RequestHeaders:
 {
  Accept: 'application/json',
  Authorization: 'Bearer api key'
} 
RequestBody:
 undefined

handleRestResponse:
 bitpanda GET https://api.exchange.bitpanda.com/public/v1/account/orders?instrument_code=BTC_EUR&with_cancelled_and_rejected=true 200 OK
ResponseHeaders:
 {
  'Bp-Remaining-Quota': '199',
  'Bp-Request-Id': '123123123123123',
  'Cache-Control': 'no-store',
  'Cf-Cache-Status': 'DYNAMIC',
  'Cf-Ray': '7124cef0db0c7897-VIE',
  Connection: 'keep-alive',
  'Content-Encoding': 'gzip',
  'Content-Type': 'application/json',
  Date: 'Sat, 28 May 2022 06:07:09 GMT',
  'Expect-Ct': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"',
  Pragma: 'no-cache',
  'Referrer-Policy': 'no-referrer',
  Server: 'cloudflare',
  'Set-Cookie': '__cf_bm=cookiedata; path=/; expires=Sat, 28-May-22 06:37:09 GMT; domain=.bitpanda.com; HttpOnly; Secure; SameSite=None, __cfruid=cookiedata
4b5ff0a1c-1653718029; path=/; domain=.bitpanda.com; HttpOnly; Secure; SameSite=None',
  'Strict-Transport-Security': 'max-age=15552000; includeSubDomains; preload',
  'Transfer-Encoding': 'chunked',
  'X-Content-Type-Options': 'nosniff',
  'X-Frame-Options': 'sameorigin',
  'X-Xss-Protection': '1; mode=block'
}
ResponseBody:
 {"order_history":[{"order":{"order_id":"orderid","account_holder":"accid","account_id":"accid","instrument_code":"BTC_EUR","time":"2022-05-27T11:57:02.613923Z","side":"BUY","price":"27000.07","amount":"0.00111","filled_a
mount":"0.00111","type":"MARKET","sequence":6728267054,"status":"FILLED_FULLY","average_price":"27000.07","is_post_only":false,"order_book_sequence":835505004,"time_last_updated":"2022-05-27T11:57:02.993985Z","update_modification_sequence":835505004},"trades":[{"fee":{"fee_amount":"0.00000166","fee_currency":"B
TC","fee_percentage":"0.15","fee_group_id":"default","fee_type":"TAKER","running_trading_volume":"0.0","collection_type":"STANDARD"},"trade":{"trade_id":"tradeid","order_id":"orderid","account_holder":"accid","account_id":"
accid","amount":"0.00111","side":"BUY","instrument_code":"BTC_EUR","price":"27000.07","time":"2022-05-27T11:57:02.810972Z","price_tick_sequence":0,"sequence":6728267054}}]}],"max_page_size":100}

/var/nodejs/trade-ts/node_modules/ccxt/js/base/Exchange.js:2119
                    reducedFees[i]['rate'] = this.parseNumber (reducedFees['i']['rate'])
                                                                               ^

TypeError: Cannot read properties of undefined (reading 'rate')
    at bitpanda.safeOrder (/var/nodejs/trade-ts/node_modules/ccxt/js/base/Exchange.js:2119:80)
    at bitpanda.parseOrder (/var/nodejs/trade-ts/node_modules/ccxt/js/bitpanda.js:1400:21)
    at /var/nodejs/trade-ts/node_modules/ccxt/js/base/Exchange.js:1561:70
    at Array.map (<anonymous>)
    at bitpanda.parseOrders (/var/nodejs/trade-ts/node_modules/ccxt/js/base/Exchange.js:1561:36)
    at bitpanda.fetchOpenOrders (/var/nodejs/trade-ts/node_modules/ccxt/js/bitpanda.js:1697:21)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async bitpanda.fetchClosedOrders (/var/nodejs/trade-ts/node_modules/ccxt/js/bitpanda.js:1704:16)
    at async /var/nodejs/trade-ts/error.js:15:17 ```


thanks

**Comment by kroitor** [comment]
@Doccos ok, that was a typo-bug, thanks for pointing it out! We fixed it in CCXT 1.84.25, the new build will arrive in ~20 minutes. Please, let us know if that solves the issue for you or not. Thank you for your feedback!

**Comment by Doccos** [comment]
@kroitor  first tests looks good, no error anymore and i got fees back.

Thank you very much
